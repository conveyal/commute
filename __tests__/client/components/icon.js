/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'

import Icon from '../../../client/components/icon'

describe('Container > Icon', () => {
  it('renders correctly', () => {
    const props = {
      type: 'pencil',
      className: 'test'
    }
    // mount component
    const tree = mount(
      <Icon
        {...props}
        />
    )
    expect(mountToJson(tree)).toMatchSnapshot()
  })
})
