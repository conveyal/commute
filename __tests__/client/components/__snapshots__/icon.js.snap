// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container > Icon renders correctly 1`] = `
<Icon
  className="test"
  type="pencil"
>
  <i
    className="fa fa-pencil fa-fw test"
  />
</Icon>
`;
